src/org/labkey/test/components/ui/entities/ParentEntityEditPanel.java

```diff
@@ -213,9 +208,7 @@ public String clickSaveExpectingError()
     public boolean isSaveButtonEnabled()
     {
         // If element is enabled the disabled attribute will not be there and getAttribute will return null.
-        return null == elementCache()
-                .button("Save")
-                .getAttribute("disabled");
+        return null == elementCache().saveButton.getAttribute("disabled");
```

Comment:
This should be able to use the standard `isEnabled` method.
```suggestion
        return ! elementCache().saveButton.isEnabled();
```